packages/components/src/vue/components/morph-parts/inflections-list.vue

```diff
@@ -230,4 +276,37 @@
 
     }
   }
+
+  .alpheios-inflections-list__pointer {
+      &-tooltip {
+          display: block;
+          height: 22px;
+          margin-left: -5px;
+      }
+
+      &-icn {
+          display: block;
+          fill: var(--alpheios-color-vivid);
+          height: 22px;
+      }
+  }
+
+  .alpheios-inflections-list__dsmbg-providers {
+      display: block;
+      height: 22px;
+
+      &-tooltip {
+        margin-left: 5px;
+      }
+
+      &-icn {
+          fill: var(--alpheios-color-neutral-dark);
+          display: block;
+          height: 22px;
+      }
+  }
+
+  .alpheios-inflections-list__dupe-infl-set {
+      display: none;
+  }
```

Comment:
it may be a site preference whether or not to show the duplicate inflection set. I think the default is not to, but I would like embedding code to be able to override this. Probably we need to expand the treebank data properties to support specification of display preferences, but that was more than I want to do in an incremental release.